.github/workflows/release.yml

```diff
@@ -166,7 +163,9 @@ jobs:
           mkdir .cargo
           cat > .cargo/config.toml <<EOF
           [target.${{ matrix.config.target }}]
-          rustflags = ["-Ctarget-cpu=generic", "-Clto", "-Cembed-bitcode=yes"]
+          rustflags = ["-C", "target-cpu=generic"]
+          [profile.release]
+          lto = "thin"
```

Comment:
we could also just add this in the `Cargo.toml`, is it better to just do it here?

Reply:
Tbh I just picked the commit we had on jormungandr